src/mantine-core/src/components/Select/Select.tsx

```diff
@@ -146,6 +157,8 @@ export const Select = factory<SelectFactory>((_props, ref) => {
     onChange,
   });
 
+  const optionsLockup = useMemo(() => getOptionsLockup(parsedData), [parsedData, _value]);
```

Comment:
Is calling this "Lockup" a mistake? Seems like it should be named "Lookup", I was going to change it, but thought I'd ask first.